docs/Antora/modules/howto_guides/pages/howto_configure_streaming.adoc

```diff
@@ -0,0 +1,298 @@
+= Configure Streaming
+
+Typically, in addition to publishing its own structure (xref:glossary:glossary.adoc#signal[Signal]s,
+xref:glossary:glossary.adoc#channel[Channel]s, xref:glossary:glossary.adoc#function_block[Function Block]s, etc.),
+an openDAQ(TM) xref:glossary:glossary.adoc#device[Device] also publishes information about
+xref:glossary:glossary.adoc#streaming[Streaming] protocols it supports. This information includes the
+streaming protocol ID (e.g. 'daq.wss' or 'daq.ns'), which doubles as a prefix in the connection string,
+enabling the routing of streaming instantiation to the appropriate openDAQ(TM)
+xref:glossary:glossary.adoc#module[Module]. Additionally, it includes the device's IP address (used if the
+streaming protocol requires a network connection) and a range of optional parameters (e.g. port number).
+When a streaming server starting on the device, this automatically generates and prepares mentioned
+information to be published along with the device's structural details.
```

Comment:
It'd be good to mention both in the how-to and the background article how the structure and streaming modules interact with each other in a separate section. Since from this it's a bit unclear on why we're using "daq.opcua" and configuring streaming when connecting to a device with that prefix, when above we only mentioned "daq.ws" and "daq.ns".

Especially since we'll be making this even more complex with the addition of structure configuration modules (eg. native) in the future.

Reply:
Section regarding modules interaction added to background info article. In how-to document: added section illustrating server modules dependency and interactions; added more detailed clarification of how client modules interact with each other.